docs/section11/Rule11-6.md

```diff
@@ -0,0 +1,40 @@
+# HVAC_SystemZoneAssignment - Rule 11-6
+
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 11-6  
+**Rule Description:** Piping losses shall not be modeled. 
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, f
```

Comment:
This has a differnt App G reference dependign on whether you are referring to the baseline or proposed design model. It looks like this check is for the baseline model whch would be Table G3.1 #11, baseline column, i.

Is there a separate RDS for checking the proposed?

Reply:
thanks.  Yes.  11-7 refers to the proposed.